x-pack/test/api_integration/apis/cloud_security_posture/helper.ts

```diff
@@ -145,6 +145,117 @@ export async function createPackagePolicy(
   return postPackageResponse.item;
 }
 
+export async function createCloudDefendPackagePolicy(
+  supertest: SuperTestAgent,
+  agentPolicyId: string,
+  roleAuthc?: RoleCredentials,
+  internalRequestHeader?: { 'x-elastic-internal-origin': string; 'kbn-xsrf': string }
+) {
+  const version = '1.2.5';
+
+  const { body: postPackageResponse } =
```

Comment:
can we move parts which are same to own variables ? eg. the object inside `send` looks pretty much the same in both code branches, probably can be extracted

Reply:
Nice catch, done.